File: python/applications/stickstick/routing.py

document.execute('defaults/application/routing/')

from com.threecrickets.prudence.util import CssUnifyMinifyFilter, JavaScriptUnifyMinifyFilter

router.capture(resources_base_url + 'data/note/{id}/', 'data/note/')
router.hide('data/note/')

# Wrap the static web with unify-minify filters
css_filter = CssUnifyMinifyFilter(None, File(application_base_path + static_web_base_path), dynamic_web_minimum_time_between_validity_checks)
java_script_filter = JavaScriptUnifyMinifyFilter(None, File(application_base_path + static_web_base_path), dynamic_web_minimum_time_between_validity_checks)
router.filterBase(static_web_base_url, css_filter, static_web)
router.filterBase(static_web_base_url, java_script_filter, css_filter)

File: javascript/applications/stickstick/routing.js
//
// Stickstick Routing
//

document.execute('/defaults/application/routing/')

importClass(
	com.threecrickets.prudence.util.CssUnifyMinifyFilter,
	com.threecrickets.prudence.util.JavaScriptUnifyMinifyFilter)

router.captureAndHide(resourcesBaseURL + 'data/note/{id}/', '/data/note/')

// Wrap the static web with unify-minify filters
var cssFilter = new CssUnifyMinifyFilter(null, new File(applicationBasePath + staticWebBasePath), dynamicWebMinimumTimeBetweenValidityChecks)
var javaScriptFilter = new JavaScriptUnifyMinifyFilter(null, new File(applicationBasePath + staticWebBasePath), dynamicWebMinimumTimeBetweenValidityChecks)
router.filterBase(staticWebBaseURL, cssFilter, staticWeb)
router.filterBase(staticWebBaseURL, javaScriptFilter, cssFilter)
